src/modules/browse.js

```diff
@@ -252,6 +258,7 @@ class Browse {
    * @param {string[]} [parameters.hiddenFacets] - Hidden facet fields to return
    * @param {object} [parameters.variationsMap] - The variations map object to aggregate variations. Please refer to https://docs.constructor.io/rest_api/variations_mapping for details
    * @param {object} [parameters.preFilterExpression] - Faceting expression to scope search results. Please refer to https://docs.constructor.io/rest_api/collections/#add-items-dynamically for details
+   * @param {object} [parameters.qsParam] - Parameters listed above can be serialized into a JSON object and parsed through this parameter. Please refer to https://docs.constructor.io/rest_api/browse/queries for details
```

Comment:
Not sure why `qs` isn't documented in the [browse queries doc](https://docs.constructor.io/rest_api/browse/queries) 🤔 

It is supported. Created a story [here](https://linear.app/constructor/issue/CSL-2280/docs-update-browse-queries-to-include-qs) to update our docs